### PR TITLE
Add details on JSON schema processing

### LINKED ISSUE #171: Add text describing implementation's ability to auto-select an applicable JSON Schema version
Since each type (`JsonSchema2023`, `CredentialSchema2023`) can apply to different versions of JSON schema we should describe the existing ability of many libraries, and the expected behavior, which is to switch on the `$schema` property and apply the right validator.

### PR DESCRIPTION
**Pull request by decentralgabe**
Fix #171

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/pull/175.html" title="Last updated on Jul 13, 2023, 11:08 PM UTC (c5eeb2f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/175/6cfae46...c5eeb2f.html" title="Last updated on Jul 13, 2023, 11:08 PM UTC (c5eeb2f)">Diff</a>

### CODE REVIEW

**Review by andresuribe87 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -423,8 +423,20 @@ <h3>Representations of JSON Schema</h3>
...
+          This section details instructions on how to process Credential Schemas, which is commonly referred to
+          as <i>JSON schema validation</i>.
         <p>
+        <p>
+         There are many open source implementations of [[JSON-SCHEMA]] validators across many common programming
+         languages. The <a href="https://openjsf.org/">OpenJS Foundation</a> maintains a list of implementations
+         as a part of the <a href="https://json-schema.org/implementations.html">JSON Schema official documentation</a>.
+        </p>
+        <p>
+         A common feature of a JSON schema validator is the ability detect the version of a JSON schema document
```
Comment (TallTed) [inline index.html:435]:
```suggestion
         A common feature of a JSON schema validator is the ability to detect the version of a JSON schema document
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -423,8 +423,20 @@ <h3>Representations of JSON Schema</h3>
...
+          This section details instructions on how to process Credential Schemas, which is commonly referred to
+          as <i>JSON schema validation</i>.
         <p>
+        <p>
+         There are many open source implementations of [[JSON-SCHEMA]] validators across many common programming
+         languages. The <a href="https://openjsf.org/">OpenJS Foundation</a> maintains a list of implementations
+         as a part of the <a href="https://json-schema.org/implementations.html">JSON Schema official documentation</a>.
+        </p>
+        <p>
+         A common feature of a JSON schema validator is the ability to detect the version of a JSON schema document
```
Comment (OR13) [inline index.html:435]: this section is normative, you probably need either cite how this is done, or provide an example in an informative section

File: index.html [hunk 3]
```diff
@@ -423,8 +423,20 @@ <h3>Representations of JSON Schema</h3>
...
+         and select the validator for that specific version of [[JSON-SCHEMA]]. It is RECOMMENDED that implementers
```
Comment (OR13) [inline index.html:436]: this is the only normative statement in this section, it feels like the whole thing could be informative.

**Review by OR13 — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (decentralgabe) [reply index.html:435]: I added some more language to clarify how it is done, and added a MUST. let me know what you think

File: index.html [hunk 4]
```diff
@@ -423,8 +423,22 @@ <h3>Representations of JSON Schema</h3>
       <section class="normative">
         <h2>Processing</h2>
         <p>
-          This section details instructions on how to process Credential Schemas.
+          This section details instructions on how to process Credential Schemas, which is commonly referred to
+          as <i>JSON schema validation</i>.
```
Comment (TallTed) [inline index.html:427]:
```suggestion
          This section provides detailed instructions on processing Credential Schemas, which is commonly referred to
          as <i>JSON schema validation</i>.
```

Comment (TallTed) [reply index.html:427]: ... or ...
```suggestion
          This section details how to process Credential Schemas, which is commonly referred to
          as <i>JSON schema validation</i>.
```

File: index.html [hunk 5]
```diff
@@ -423,8 +423,22 @@ <h3>Representations of JSON Schema</h3>
...
         <p>
+        <p>
+         There are many open source implementations of [[JSON-SCHEMA]] validators across many common programming
+         languages. The <a href="https://openjsf.org/">OpenJS Foundation</a> maintains a list of implementations
+         as a part of the <a href="https://json-schema.org/implementations.html">JSON Schema official documentation</a>.
+        </p>
+        <p>
+         A common feature of a JSON schema validator is the ability to detect the version of a JSON schema document
+         and select the validator for that specific version of [[JSON-SCHEMA]]. This is done by switching on the
+         schemas <code>$schema</code> and picking the corresponding validator. Schemas without a <code>$schema</code>
```
Comment (TallTed) [inline index.html:437]:
```suggestion
         schema's <code>$schema</code> and picking the corresponding validator. Schemas without a <code>$schema</code>
```

File: index.html [hunk 6]
```diff
@@ -423,8 +423,22 @@ <h3>Representations of JSON Schema</h3>
...
+         property are not to be considered valid and MUST not be processed. It is RECOMMENDED that implementers
```
Comment (TallTed) [inline index.html:438]:
```suggestion
         property are not considered valid and MUST not be processed. It is RECOMMENDED that implementers
```

File: index.html [hunk 7]
```diff
@@ -423,8 +423,22 @@ <h3>Representations of JSON Schema</h3>
...
+         choose validators which possess this capability, and are able to limit validation to the
```
Comment (TallTed) [inline index.html:439]:
```suggestion
         choose validators which possess this capability and are able to limit validation to the 
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor tweaks

**Comment by decentralgabe** [comment]
Merging; open to revising later.

**Pull request merged**